core-backend/sandbox/src/funcs.rs

```diff
@@ -940,6 +939,7 @@ where
                     let length = match res {
                         Ok(error) => {
                             ctx.write_output(error_bytes_ptr, error.as_ref())?;
+                            ctx.ext.charge_error().map_err(RuntimeCtxError::Ext)?;
```

Comment:
For now when we don't have per_byte dependency for gr_error, charging should always appear, despite the result of `res`